bip-octojoin.mediawiki

```diff
@@ -0,0 +1,111 @@
+  BIP: XXX
+  Layer: Applications
+  Title: Octojoin v1
+  Author: /dev/fd0 <alicexbt@protonmail.com>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0xxx
+  Status: Draft
+  Type: Informational
+  Created: 2024-09-25
+  License: BSD-2-Clause
+
+
+==Abstract==
+
+This document describes the process to use swapped UTXOs for inputs, [[bip-0352.mediawiki|BIP 352]] silent payment addresses for outputs, and hide the transacted amount with multiple interpretations.
+
+==Motivation==
+
+Payjoin requires coordination and sharing of UTXOs between sender and recipient. This affects privacy and UX in some cases. Octojoin avoids sharing of UTXOs and uses swapped UTXOs.
+
+====Relation to Monero====
+
+Most Monero transactions use 16 ring size for the input, making it impossible to determine which input is signed by the user out of 16. The amounts are hidden, and stealth addresses are used for outputs.
+
+Octojoin achieves partial obfuscation similar to Monero's ring signatures ([https://www.getmonero.org/2024/04/27/fcmps.html FCMP++] will replace ring signatures in the future) and stealth addresses. Swaps can be done off-chain to obscure the transaction's input ownership and history.
+
+==Specification==
+
+====Sequence Diagram====
+
+<source lang="mermaid">
+sequenceDiagram
+    participant Alice
+    participant Wallet
+    participant Bob
+
+    Alice->>Wallet: Label swapped UTXOs as 'octojoin'
+    Bob->>Alice: Share silent payment address
+    Alice->>Wallet: Create transaction using 'octojoin' UTXOs + 1 normal input
+    Note right of Alice: Minimum 2 outputs and avoid change for multiple interpretations
+    Wallet->>Bob: Pay Bob
+</source>
+
+Users should label some UTXOs as “octojoin” so that they can be used in octojoin transactions. Ideally, these UTXOs should be ones that have been swapped with others and are linked to someone else’s transaction history.
+
+'''Input Obfuscation with Off-Chain Swaps'''
+
+Do off-chain swaps (e.g. statechain, submarine swaps, or coinswap) to obtain UTXOs with different histories. Ensure that all inputs come from other users except one belonging to the sender, making it difficult for on-chain analysts to determine ownership.
```

Comment:
- A statechain UTXO is co-owned by the statechain operator and the last recipient in a multisig. Multisig inputs do not participate in the secret derivation. 
- Submarine swaps refer to a multi-hop lightning payment whose last hop is executed as an on-chain payment to the recipient. Submarine swaps are easily identifiable via an on-chain HTLC construction and also include a multisig construction.
- A coinswap is a payment received to the recipient whose inputs were not controlled by the sender.

Given that two of the listed examples exhibit uncommon patterns, and the third constitutes a payment to a wallet that was executed by a different UTXO owner than the sender, it’s not clear to me how a transaction funded by a combination of such inputs would have significantly improved privacy properties. If an observer simply categorizes the inputs and analyses their pedigree under the corresponding context, it seems likely that a transaction would be sufficiently recognizable as an Octojoin and given the small number of Statechain and Submarine Swap providers, potentially even identifiable via information requests to such service providers. Could you please expand the motivation section by elaborating your arguments why participation in this scheme is expected to lead to a privacy improvement?